### ISSUE TITLE
ERROR ReferenceError: isString is not defined

### ISSUE THREAD

**Issue by R2-D1** [easylogic/colorpicker#36]
**Describe the bug**

When I try to run createGradientPicker with option 'gradient: 'conic-gradient(from 90deg at 50% 50%, white  0deg,black  360deg)'  - In console error `ERROR ReferenceError: isString is not defined`. 

**To Reproduce**
1. Run      
```
ColorPickerUI.createGradientPicker({
        position: "inline",
        container: document.querySelector('.test-elem'),
        gradient: 'conic-gradient(from 90deg at 50% 50%, white  0deg,black  360deg)',
      });
```

**Expected behavior**
Inited picker with correct gradient

**Screenshots**
Error in console
![Screenshot 2022-01-19 at 14 40 28](https://user-images.githubusercontent.com/46542768/150132692-b6901347-0cef-4f43-b68c-9d7c11fc161b.png)
Interface while error
![Screenshot 2022-01-19 at 14 40 18](https://user-images.githubusercontent.com/46542768/150132884-00dd6652-f40e-4490-826a-d03197ed6652.png)

**Desktop (please complete the following information):**
 - OS:  MacOS
 - Browser chrome
 - Version 97.0.4692.71

**Additional context**
String for gradient was generated by gradient picker. I can generate this gradient value without any error, but can't init picker with generated value. Error reproduce also with repeating-conic-gradient value.

**Comment by R2-D1** [comment]
Can anyone help?

**Comment by Dmitrii-Dudnyk** [comment]
Have the same issue

**Comment by R2-D1** [comment]
@easylogic Hi, do you support this project? about month without answer

**Comment by easylogic** [comment]
@R2-D1  @Dmitrii-Dudnyk   

Sorry I checked the issue too late.

I'll check and let you know right away.

**Comment by easylogic** [comment]
@R2-D1 @Dmitrii-Dudnyk 

I released v1.10.10

Please test it.

**Comment by R2-D1** [comment]
@easylogic Thanks, now it's works without syntax error, but i found another few errors
1. With angle - after init it resets to 0. For example if value was `conic-gradient(from -40.7deg at 50% 50%, rgb(49,0,255)  0deg,rgb(255,255,255)  228deg)` after init it will `conic-gradient(from 0deg at 50% 50%, rgb(49,0,255)  0deg,rgb(255,255,255)  228deg)`. It's reproduce with conic and repeating conic gradient.
2. I found problem with angle when I try to change gradient from conic to linear - gradient is broken. For example - previous gradient was `conic-gradient(from 666.3deg at 50% 50%, rgb(102,81,191)  0deg,rgb(255,255,255)  228deg)`, after change to linear it's `linear-gradient(666.3, rgb(102,81,191)  0%,rgb(255,255,255)  63.333%)` (angle is broken). Interface after this error  <img width="247" alt="Screenshot 2022-03-08 at 20 56 55" src="https://user-images.githubusercontent.com/46542768/157306261-c3d6ae8c-4a09-45a2-870a-598c20add83c.png"> (selected linear but preview is conic).

**Comment by easylogic** [comment]
@R2-D1  

Thanks for reporting. 
I'll check again and tell you

**Comment by easylogic** [comment]
@R2-D1 

I fixed an angle issue.
I released v1.10.11. 

Please test it.

**Comment by R2-D1** [comment]
@easylogic thanks a lot, looks perfect
